### ISSUE TITLE
Define target namespace separately 

### ISSUE THREAD

**Issue by haf-tech** [redhat-developer/kam#246]
**Is your feature request related to a problem? Please describe.**
in the current environment and project setting, any team will receive an own cicd namespace holding project specific CI/CD pipeline definition. The namespace has a dedicated name, let's say `prj-common-cicd`
The `kam bootstrap` command generates files for predefined namespaces, like `cicd`. It is possible to set a `--prefix`, this is not enough. Changing the namespace in the generated configuration is not possible in case `SealedSecret` with `strict` scope is used.

**Describe the solution you'd like**
Possibility to define the target namespaces for all aspects (cicd, stages, common ArgoCD resources etc)

**Describe alternatives you've considered**
n/a

**Additional context**
I did not checked the interactive mode of `kam` if this is supported?! :)

**Comment by chetan-rns** [comment]
@haf-tech I agree prefix wouldn't be sufficient. IMO there should be means to update the namespace after the bootstrap as well. One option could be updating it in the manifest file and Kam will update the namespace and the structure accordingly. I suppose `kam build` cmd was introduced for this purpose which is not ready yet.
